src/daemon/http/server.rs

```diff
@@ -1241,6 +1250,7 @@ async fn api_ca_children(req: Request, path: &mut RequestPath, ca: Handle) -> Ro
         },
         None => match *req.method() {
             Method::POST => api_ca_add_child(req, ca).await,
+            Method::GET => api_ca_children_stats(req, ca).await,
```

Comment:
It feels a bit strange to get statistics about the connections made by a child CA to us by fetching the "contact" for the child, this is not at all the inverse of the POST operation. Should stats perhaps have their own endpoint?

Reply:
Moved to dedicated endpoint. You're right.. it wasn't a good match. There was no more namespace under 'api/v1/cas/<ca>/children', because the next level maps to a specific child. But I went for `api/v1/cas/<ca>/stats/children/connections` instead. This also leaves room for extension with other stuff.